litedram/phy/ddr5/sim.py

```diff
@@ -106,8 +106,9 @@ def __init__(self, pads, data_cdc, *, clk_freq, log_level, init_delays=False):
         # Mode Registers storage
         self.mode_regs = Array([Signal(8) for _ in range(64)])
         # Active banks
-        self.active_banks = Array([Signal() for _ in range(8)])
-        self.active_rows = Array([Signal(18) for _ in range(8)])
+        self.number_of_banks = 32;
```

Comment:
I think this number should be obtained by the PHY settings

Reply:
Hmm, we might probably separate this maybe have as follows:

```
self.bankbits    = bankbits    = 5
```

and upon defining the DFI Interface add to that the additional three bits for the MRW. I do not see the bankbits used anywhere else.

Reply:
I found references by grep, but didn't check if they really refer to field of this class. It looks that you are right.
What do you think about it?
https://github.com/antmicro/litedram/pull/13/commits/9535824d7fdeeb46f4504cb3a11685a7c035c8bb#diff-a90dfa0fea0bdebae44795cca7e1841005e69b6a9e9396a73cb8a08bd7f74edaR110
If it's ok, I will use it also for row and column sizes.
I see also that these values have to be updated:
https://github.com/antmicro/litedram/blob/b2a342f767737ee6e48604941c2dd172a39eb6ae/litedram/modules.py#L1170-L1172

Reply:
I think looks good